### PR TITLE
Add a command to detect active calls

### LINKED ISSUE #3671: Command which tells admins if a call is going on
In some cases a command which tells an admin if a call is going on would be nice.
This would allow checking before applying fixes or updates.

Currently the following query can be used:
```sql
SELECT r.id, r.name, r.token, p.user_id FROM oc_talk_rooms r LEFT JOIN oc_talk_participants p ON p.room_id = r.id AND p.participant_type IN (1, 2) WHERE r.active_since IS NOT NULL;
```

### PR DESCRIPTION
**Pull request by nickvergessen**
### No calls

Exit code 0
Bildschirmfoto von 2020-05-27 11-37-15a

### Call in progress

Exit code 1
Bildschirmfoto von 2020-05-27 11-41-27

Fix #3671

### CODE REVIEW

File: lib/Command/ActiveCalls.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			->from('talk_participants')
+			->where($query->expr()->gt('in_call', $query->createNamedParameter(Participant::FLAG_DISCONNECTED)))
+			->andWhere($query->expr()->neq('session_id', $query->createNamedParameter('0')))
+			->andWhere($query->expr()->gt('last_ping', $query->createNamedParameter(time() - 60)));
+
+		$result = $query->execute();
+		$numParticipants = (int) $result->fetchColumn();
+		$result->closeCursor();
+
+		$output->writeln(sprintf('<error>There are currently %1$d calls in progress with %1$d participants</error>', [$numCalls, $numParticipants]));
```
Comment (danxuliu) [inline lib/Command/ActiveCalls.php:79]: This always prints 1 call and 1 participant, no matter the values of `$numCalls` and `$numParticipants`.

File: lib/Command/ActiveCalls.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		if ($numCalls === 0) {
+			$output->writeln('<info>No calls in progress</info>');
+			return 0;
+		}
+
+		$query->select($query->func()->count('*', 'num_participants'))
+			->from('talk_participants')
+			->where($query->expr()->gt('in_call', $query->createNamedParameter(Participant::FLAG_DISCONNECTED)))
+			->andWhere($query->expr()->neq('session_id', $query->createNamedParameter('0')))
+			->andWhere($query->expr()->gt('last_ping', $query->createNamedParameter(time() - 60)));
```
Comment (danxuliu) [inline lib/Command/ActiveCalls.php:73]: The count of participants in the call does not seem to be right; I started a call with just one participant and the count returned three, and when I started a second call the participants count went up to six. Note that is not always a multiple of three, though; at another point during the tests it returned eight participants.

File: lib/Command/ActiveCalls.php (hunk 1 above)
Comment (nickvergessen) [reply lib/Command/ActiveCalls.php]: yeah upsy

File: lib/Command/ActiveCalls.php (hunk 2 above)
Comment (nickvergessen) [reply lib/Command/ActiveCalls.php:73]: The count was correct, the user placeholder was not

**Pull request merged**